core/sequences/sequences-docs.factor

```diff
@@ -678,7 +678,7 @@ HELP: replace-slice
 { $description "Replaces a range of elements beginning at index " { $snippet "from" } " and ending before index " { $snippet "to" } " with a new sequence." }
 { $errors "Throws an error if " { $snippet "new" } " contains elements whose types are not permissible in " { $snippet "seq" } "." } ;
 
-{ push prefix suffix } related-words
+{ push push-all push-either push-if pop pop* prefix suffix suffix! } related-words
```

Comment:
I'm not sure how many related words we should have, for example ``push`` puts something at the end of a sequence, so I think ``suffix`` and ``suffix!`` make sense, but maybe ``prefix`` isn't close enough?  Not sure where the line is, but a list of all "modifies a sequence" is probably too wide.

Also ``push-all`` pushes several things, so is it more related to ``append`` and ``append!`` rather than ``push``?